docs/modules/vector-search/pages/vector-collections.adoc

```diff
@@ -0,0 +1,498 @@
+
+NOTE: At this point, each metadata object must correspond to exactly one vector in each index.
+
+NOTE: When working with multiple indexes in your collections, it's necessary to specify the index to search in your queries..
+
+== Configuration
+
+Collection configuration can be set dynamically during vector collection creation or statically during cluster configuration. Below is a table listing all available options:
+
+.Collection configuration options
+[cols="1,2,1,1",options="header"]
+|===
+|Option|Description|Required|Default
+
+|name
+|The name of the vector collection.
+|Y
+|null
+
+|indexes
+|Information about indexes configuration
+|Y
+|null
+|===
+
+.Index configuration options
+[cols="1,2,1,1",options="header"]
+|===
+|Option|Description|Required|Default
+
+|name
+|The name of the vector index.
+|`N` - for single-index vector collections. `Y` - for multi-index collection
+|null
+
+|dimension
+|Vectors dimension
+|Y
+|null
+
+|metric
+|Distance metric used to calculate the distance between two vectors.
+See table Available distance metrics.
+|Y
+|null
+
+|max-degree
+|The maximum number of neighbors per node is calculated as max-degree * 2
+|Y
+|16
+
+|ef-construction
+|The size of the search queue to use when finding nearest neighbors.
+|Y
+|100
+
+|use-deduplication
+|The option for vector deduplication.
+When disabled, a new node will be created for each vector, even if it is the same as an existing one.
+|true
+
+|===
+
+.Available distance metrics
+[cols="2,2,2",options="header"]
+|===
+|Name|Description| Score definition
+
+|EUCLIDEAN
+|euclidean distance
+|1 / (1 + squareDistance(v1, v2))
+
+|COSINE
+|cosine of the angle between the vectors
+|(1 + cos(v1, v2)) / 2
+
+|DOT
+|dot product of the vectors
+|(1 + dotProduct(v1, v2)) / 2
+
+|===
+
+WARNING: The recommended method for computing cosine similarity is to normalize all vectors to unit length and utilize the DOT metric instead.
+
+
+Configuration example:
+
+[tabs]
+====
+XML::
++
+--
+[source,xml]
+----
+<hazelcast>
+    <vector-collection name="vector-collection-name">
+        <indexes>
+            <index name="index-1">
+                <dimension>2</dimension>
+                <metric>DOT</metric>
+            </index>
+            <index name="index-2">
+                <dimension>2</dimension>
+                <metric>DOT</metric>
+                <max-degree>10</max-degree>
+                <ef-construction>20</ef-construction>
+                <use-deduplication>false</use-deduplication>
+            </index>
+        </indexes>
+    </vector-collection>
+</hazelcast>
+----
+--
+YAML::
++
+--
+[source,yaml]
+----
+hazelcast:
+  vector-collection:
+    vector-collection-name:
+      indexes:
+        - name: index-1
+          dimension: 2
+          metric: DOT
+        - name: index-2
+          dimension: 2
+          metric: DOT
+          max-degree: 10
+          ef-construction: 20
+          use-deduplication: false
+----
+--
+Java::
```

Comment:
shall we also introduce a Python sample?

Reply:
The API for Python is different from that for Java. I expected it to be in a separate document.